<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white</string>
    </property>
    <property name="text">
     <string>MentalHealthTracker</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="pages_stack">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>39</y>
      <width>801</width>
      <height>521</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_myDay">
     <widget class="QFrame" name="submenu_myDay">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>121</width>
        <height>521</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(91, 91, 91)</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="btn_edit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>173</y>
         <width>121</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: white</string>
       </property>
       <property name="text">
        <string>Edit Day</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_addEntry">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>110</y>
         <width>121</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: white</string>
       </property>
       <property name="text">
        <string>Add Entry</string>
       </property>
      </widget>
     </widget>
     <widget class="QStackedWidget" name="substack_myDay">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>0</y>
        <width>681</width>
        <height>521</height>
       </rect>
      </property>
      <widget class="QWidget" name="subpage_newDay">
       <widget class="QTextEdit" name="entry_log">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>260</y>
          <width>611</width>
          <height>171</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_rating">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>221</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>How would you rate your day so far?</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_emotions">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>110</y>
          <width>181</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>What are your current emotions?</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_log">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>220</y>
          <width>201</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>What's happened today?</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_submit">
        <property name="geometry">
         <rect>
          <x>284</x>
          <y>443</y>
          <width>101</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Submit</string>
        </property>
       </widget>
       <widget class="QSlider" name="slider_rating">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>70</y>
          <width>601</width>
          <height>22</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
        <property name="sliderPosition">
         <number>5</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEntry_emotion">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>160</y>
          <width>171</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QListWidget" name="list_emotions">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>120</y>
          <width>331</width>
          <height>101</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_emotionLine">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>150</y>
          <width>60</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Emotion:</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spin_severity">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>190</y>
          <width>42</width>
          <height>22</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_emotionSeverity">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>180</y>
          <width>60</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Severity:</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="subpage_thanks">
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>180</y>
          <width>131</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Thank you for sharing!</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>240</y>
          <width>301</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>Come back tomorrow to keep up with your log!</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="subpage_edit"/>
      <widget class="QWidget" name="subpage_log">
       <widget class="QTextEdit" name="entry_logUpdate">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>110</y>
          <width>621</width>
          <height>321</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_updateLog">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>450</y>
          <width>75</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Update Log</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_logUpdate">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>80</y>
          <width>171</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>What's been happening?</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_help"/>
    <widget class="QWidget" name="page_resources"/>
    <widget class="QWidget" name="page_settings">
     <widget class="QFrame" name="submenu_settings">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>121</width>
        <height>521</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(91, 91, 91)</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="btn_notifications">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>173</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: white</string>
       </property>
       <property name="text">
        <string>Notifications</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_colors">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>130</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: white</string>
       </property>
       <property name="text">
        <string>Colors</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_resetUserData">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>450</y>
         <width>121</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: white</string>
       </property>
       <property name="text">
        <string>Reset User Data</string>
       </property>
      </widget>
     </widget>
     <widget class="QStackedWidget" name="substack_settings">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>0</y>
        <width>681</width>
        <height>441</height>
       </rect>
      </property>
      <widget class="QWidget" name="subpage_notifications">
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>641</width>
          <height>401</height>
         </rect>
        </property>
        <property name="title">
         <string>Notifications</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="subpage_thanks_2">
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>180</y>
          <width>131</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Thank you for sharing!</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>240</y>
          <width>301</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>Come back tomorrow to keep up with your log!</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="subpage_colors">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>641</width>
          <height>401</height>
         </rect>
        </property>
        <property name="title">
         <string>Colors/Design</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="btn_applyChanges">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>440</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Apply Changes</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_resetToDefaults">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>440</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Reset to Defaults</string>
      </property>
     </widget>
     <zorder>substack_settings</zorder>
     <zorder>submenu_settings</zorder>
     <zorder>btn_applyChanges</zorder>
     <zorder>btn_resetToDefaults</zorder>
    </widget>
    <widget class="QWidget" name="page_graphing">
     <widget class="QFrame" name="submenu_myDay_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>121</width>
        <height>521</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(91, 91, 91)</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="btn_addEntry_2">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>120</y>
         <width>121</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: white</string>
       </property>
       <property name="text">
        <string>Type</string>
       </property>
      </widget>
     </widget>
     <widget class="QStackedWidget" name="substack_myDay_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>0</y>
        <width>681</width>
        <height>521</height>
       </rect>
      </property>
      <widget class="QWidget" name="subpage_newDay_2">
       <widget class="QLabel" name="label_rating_2">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>110</y>
          <width>221</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Please choose a graph type:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>240</y>
          <width>75</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Line</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>240</y>
          <width>75</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Bar</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>240</y>
          <width>75</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Scatter</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(67, 67, 67)</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="btn_myDay">
     <property name="geometry">
      <rect>
       <x>174</x>
       <y>0</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white</string>
     </property>
     <property name="text">
      <string>My Day</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_graphing">
     <property name="geometry">
      <rect>
       <x>284</x>
       <y>0</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white</string>
     </property>
     <property name="text">
      <string>Graphing</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_settings">
     <property name="geometry">
      <rect>
       <x>394</x>
       <y>0</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white</string>
     </property>
     <property name="text">
      <string>Settings</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_help">
     <property name="geometry">
      <rect>
       <x>504</x>
       <y>0</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white</string>
     </property>
     <property name="text">
      <string>Help</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_resources">
     <property name="geometry">
      <rect>
       <x>614</x>
       <y>0</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white</string>
     </property>
     <property name="text">
      <string>Resources</string>
     </property>
    </widget>
   </widget>
   <zorder>pages_stack</zorder>
   <zorder>frame</zorder>
   <zorder>label</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExport"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Options</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>btn_myDay</tabstop>
  <tabstop>btn_graphing</tabstop>
  <tabstop>btn_settings</tabstop>
  <tabstop>btn_help</tabstop>
  <tabstop>btn_resources</tabstop>
  <tabstop>slider_rating</tabstop>
  <tabstop>lineEntry_emotion</tabstop>
  <tabstop>list_emotions</tabstop>
  <tabstop>entry_log</tabstop>
  <tabstop>btn_submit</tabstop>
  <tabstop>btn_addEntry</tabstop>
  <tabstop>btn_edit</tabstop>
  <tabstop>btn_updateLog</tabstop>
  <tabstop>entry_logUpdate</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
